core/src/test/java/brooklyn/location/geo/HostGeoLookupIntegrationTest.java

```diff
@@ -34,6 +35,7 @@
 
     public static final Logger log = LoggerFactory.getLogger(HostGeoLookupIntegrationTest.class);
     
+    // Needs fast network connectivity to figure out the external IP. If response not returned in 2s fails.
```

Comment:
Interesting. Longer term do you think it's right to fail? Is it possible to differentiate between "still waiting" and failed? Could we wait longer? Or could we throw `SkipException` if we know that it's not a true failure?

Certainly happy with as-is for this release.